src/pages/search/index.tsx

```diff
@@ -365,90 +374,34 @@ const Search: InferGetServerSidePropsType<typeof getServerSideProps> = ({ theme,
-                <>
-                  <SearchFilters
-                    searchCriteria={searchQuery}
-                    query={router.query}
-                    regions={regions}
-                    countries={countries}
-                    corpus_types={corpus_types}
-                    conceptsData={conceptsData}
-                    handleFilterChange={handleFilterChange}
-                    handleYearChange={handleYearChange}
-                    handleRegionChange={handleRegionChange}
-                    handleClearSearch={handleClearSearch}
-                    handleDocumentCategoryClick={handleDocumentCategoryClick}
-                    featureFlags={featureFlags}
-                  />
-                </>
-              )}
-            </div>
-            <div className="mt-4 text-xs" data-cy="number-of-results">
-              {status === "success" && <NoOfResults hits={hits} queryString={qQueryString} />}
-            </div>
-          </SiteWidth>
-          {/* END MOBILE ONLY */}
           <MultiCol id="search">
-            <SideCol extraClasses="hidden md:block relative">
+            <SideCol
+              extraClasses={`absolute z-99 top-0 w-screen bg-white duration-250 ease-[cubic-bezier(0.04, 0.62, 0.23, 0.98)] ${
+                showFilters ? "translate-y-[0%]" : "fixed translate-y-[100vh]"
+              } md:translate-y-[0%] md:h-full md:sticky md:top-[72px] md:z-50`}
+            >
               {configQuery.isFetching ? (
                 <Loader size="20px" />
               ) : (
                 <>
-                  <div className="sticky top-[72px] z-50">
-                    <div className="z-10 px-5 bg-white border-r border-gray-300 pt-5 sticky top-0 h-screen overflow-y-auto scrollbar-thumb-gray-200 scrollbar-thin scrollbar-track-white scrollbar-thumb-rounded-full hover:scrollbar-thumb-gray-500">
-                      <SearchFilters
-                        searchCriteria={searchQuery}
-                        query={router.query}
-                        regions={regions}
-                        countries={countries}
-                        corpus_types={corpus_types}
-                        conceptsData={conceptsData}
-                        handleFilterChange={handleFilterChange}
-                        handleYearChange={handleYearChange}
-                        handleRegionChange={handleRegionChange}
-                        handleClearSearch={handleClearSearch}
-                        handleDocumentCategoryClick={handleDocumentCategoryClick}
-                        featureFlags={featureFlags}
-                      />
-                    </div>
-                    <SlideOut showCloseButton={false}>
+                  <div className="sticky md:top-[72px] h-screen md:h-[calc(100vh-72px)] px-5 bg-white border-r border-gray-300 pt-5 pb-[70px] overflow-y-auto scrollbar-thumb-gray-200 scrollbar-thin scrollbar-track-white scrollbar-thumb-rounded-full hover:scrollbar-thumb-gray-500 md:pb-0">
```

Comment:
Question: Is `calc(100vh-72px)` still relevant here after my change made the navbar sticky again?

Reply:
Yeh otherwise it was trying to stick underneath the header.